External/Plugins/ProjectManager/Actions/FileActions.cs

```diff
@@ -445,34 +445,29 @@ public bool Rename(string oldPath, string newName)
 
                 if (isDirectory)
                 {
-                    // this is required for renaming directories, don't ask me why
-                    string oldPathFixed = (oldPath.EndsWith('\\')) ? oldPath : oldPath + "\\";
-                    string newPathFixed = (newPath.EndsWith('\\')) ? newPath : newPath + "\\";
-                    if (oldPathFixed.Equals(newPathFixed, StringComparison.OrdinalIgnoreCase))
```

Comment:
It used to be because of `FileHelper.ForceMoveDirectory()`, [this line](https://github.com/fdorg/flashdevelop/pull/1291/files#diff-babcd5980fed43a8a6c7f6fccae1d1afR228).

However since the method now seems to add +1 to the length variable if the backslash isn't present, so it's not an issue anymore.